tf/chunkparser.py

```diff
@@ -278,24 +388,36 @@ def convert_v5_to_tuple(self, content):
                  self.flat_planes[1]
 
         assert len(planes) == ((8 * 13 * 1 + 8 * 1 * 1) * 8 * 8 * 4)
-        winner = float(winner)
-        assert winner == 1.0 or winner == -1.0 or winner == 0.0
-        winner = struct.pack('fff', winner == 1.0, winner == 0.0,
+
+        # jhorthos query - pls check this
+        if version == V6_VERSION:
+            winner = struct.pack('fff', (1.0 - result_d + result_q)/2, result_d,
```

Comment:
For consistency with below - rather than `/2` maybe `* 0.5`

Reply:
ok